runtime/src/main/java/com/redhat/ipaas/rtdb/SQLRealTimeDB.java

```diff
@@ -0,0 +1,270 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.rtdb;
```

Comment:
Could move it around.  Was thinking it's runtime's choice of which impl of RealTimeDB to use.  We could build one on a real NoSQL db one day.